jenkins/ruby/distribution-build.jenkinsfile

```diff
@@ -40,8 +40,10 @@ pipeline {
                         writeFile file: 'gem-private_key.pem', text: RUBYGEMS_PRIVATE_KEY
                     }
                     sh'''
+                        [[ -s "/usr/local/rvm/scripts/rvm" ]] && source "/usr/local/rvm/scripts/rvm"
```

Comment:
Does this work if we disable running as root or did we need to update the docker image?

Reply:
I don't think so, but I haven't tested it yet. We installed rvm as root user so we will need root access to install other interpreters. In this way, we don't have to modify our docker image.